src/index.ts

```diff
@@ -1,56 +1,298 @@
+      })
+    },
+    createReview: (_parent, reviewInput: Prisma.UserReviewCreateInput) => {
+      return prisma.userReview.create({
+        data: {
+          createdAt: reviewInput.createdAt,
+          reputation: reviewInput.reputation,
+          ReviewData: reviewInput.ReviewData,
+          user: reviewInput.user,
+          Business: reviewInput.Business
+        }
+      })
+    },
+    updateUser: (_parent, args: {id: number, userInput: Prisma.UserUpdateInput}) => {
+      return prisma.user.update({
+        data: {
+          firstName: args.userInput.firstName,
+          lastName: args.userInput.lastName,
+          email: args.userInput.email,
+          password: args.userInput.password,
+          Comment: {},
+          UserReview: {},
+          // below here are optional (do i add them or not?)
+          roles: args.userInput.roles,
+          profilePhoto: args.userInput.profilePhoto,
+          Business: args.userInput.Business
+        },
+        where: {id: args.id}
+      })
+    },
+    updateBusiness: (_parent, args: {id: number, businessInput: Prisma.BusinessUpdateInput}) => {
+      return prisma.business.update({
+        data: {
+          name: args.businessInput.name,
+          sustainabilityScore: args.businessInput.sustainabilityScore,
+          customerScore: args.businessInput.customerScore,
+          owner: args.businessInput.owner,
+          createdAt: args.businessInput.createdAt,
+          updatedAt: args.businessInput.updatedAt,
+
+          ProfileText: args.businessInput.ProfileText,
+          ProfilePicture: args.businessInput.ProfilePicture,
+          type: args.businessInput.type,
+          locationId: args.businessInput.locationId,
+          primaryLocation: args.businessInput.primaryLocation,
+          comments: args.businessInput.comments,
+          reviews: args.businessInput.reviews
+        },
+        where: {id: args.id}
+      })
+    },
+    updateLocation: (_parent, args: {id: number, locationInput: Prisma.LocationUpdateInput}) => {
+      return prisma.location.update({
+        data: {
+          address: args.locationInput.address,
+          createdAt: args.locationInput.createdAt,
+          updatedAt: args.locationInput.updatedAt,
+
+          longitude: args.locationInput.longitude,
+          latitude: args.locationInput.latitude,
+          businessName: args.locationInput.businessName,
+          Business: args.locationInput.Business,
+          Comment: args.locationInput.Comment
+        },
+        where: {id: args.id}
+      })
+    },
+    updateComment: (_parent, args: {id: number, commentInput: Prisma.CommentUpdateInput}) => {
+      return prisma.comment.update({
+        data: {
+          text: args.commentInput.text,
+          createdAt: args.commentInput.createdAt,
+          location: args.commentInput.location,
+          user: args.commentInput.user,
+
+          business: args.commentInput.business
+        },
+        where: {id: args.id}
```

Comment:
This can be simplified using the spread operator

Reply:
Might be possible, but the first and last words on each line (e.g. text) refer to different variables. The first word is a property of the existing object and the second is from the new object (e.g. reviewInput) being passed as a parameter and contains the new values.